src/renderer/assets/images/pages/contact-new/purse.svg

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we call this `wallet.svg` instead

Reply:
Smells like @olejegcord in here.

Reply:
![image](https://user-images.githubusercontent.com/8069294/51616776-ba82e380-1f22-11e9-8764-116aaa6ae500.png)

Reply:
![image](https://user-images.githubusercontent.com/8069294/51616799-c7073c00-1f22-11e9-9894-d1147489e270.png)

Reply:
The initial file that I got was called `images.svg` or something alike, so I pay my own tribute to Oleg, calling it `purse.svg` to make it less confusing than calling it `wallet.svg`, which could be interpreted as a background for wallets, not for contacts only.

I've renamed it to `wallets.svg`, @alexbarnsley .